subprojects/logging/src/integTest/groovy/org/gradle/DeprecatedUsageBuildOperationProgressIntegrationTest.groovy

```diff
@@ -70,10 +70,19 @@ class DeprecatedUsageBuildOperationProgressIntegrationTest extends AbstractInteg
                 }
             }
 
+            task t3 {
+                doLast {
+                    def latch = new java.util.concurrent.CountDownLatch(1)
```

Comment:
seems to be unused? Was this going to be used to synchronise with the other thread?

Reply:
Forgot to use it. Added.